scripts/shared/lib/deploy_operator

```diff
@@ -33,10 +33,12 @@ function deploytool_prereqs() {
 }
 
 function setup_broker() {
+    gn=
+    [[ $globalnet = true ]] && gn=--globalnet
```

Comment:
```suggestion
    [[ $globalnet != true ]] || gn="--globalnet"
```

Reply:
This is very important, in order to avoid exiting bash scripts (that has set -e) when **evaluating false conditions**. 
We should replace all these conditions: `[[ condition ]] && command`
With:   `[[ ! condition ]] || command`

Reply:
This is very important, in order to avoid exiting bash scripts (that has set -e) when **evaluating false conditions**.
We should replace all these conditions: `[[ condition ]] && command`
With: `[[ ! condition ]] || command`

Reply:
Done